### PR TITLE
fix: update sha1 to sha256 and other chores

### PR DESCRIPTION
**Pull request by priteshbandi**
Fixes: https://github.com/notaryproject/notation/issues/611 and https://github.com/notaryproject/notation/issues/635

### CODE REVIEW

File: cmd/notation/inspect.go [hunk 1]
```diff
@@ -269,7 +272,7 @@ func printOutput(outputFormat string, ref string, output inspectOutput) error {
 
 		certListNode := sigNode.Add("certificates")
 		for _, cert := range signature.Certificates {
-			certNode := certListNode.AddPair("SHA1 fingerprint", cert.SHA1Fingerprint)
+			certNode := certListNode.AddPair("SHA256 fingerprint", cert.SHA256Fingerprint)
```
Comment (JeyJeyGao) [inline cmd/notation/inspect.go:275]: Do we need to update the spec?

Comment (priteshbandi) [reply cmd/notation/inspect.go:275]: Good catch, updated.

**Comment by Two-Hearts** [comment]
By searching `Sha1` in Notation CLI, I found we are also showing `Sha1` in the `notation cert show` command: https://github.com/notaryproject/notation/blob/5516199cd2d83349468682379c1e1f4ccd24414a/cmd/notation/internal/truststore/truststore.go#L124
Should we also change it to `Sha2` in this PR?

**Review by Two-Hearts — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by rgnote** [comment]
LGTM

**Review by rgnote — APPROVED** [review]

**Pull request merged**